xinference/model/llm/utils.py

```diff
@@ -565,7 +566,14 @@ def _eval_tool_arguments(cls, model_family, c):
         return result
 
     @classmethod
-    def _tool_calls_completion_chunk(cls, model_family, model_uid, c, chunk_id=None):
+    def _tool_calls_completion_chunk(
```

Comment:
This function is not only for tool call, maybe we can call it `_post_process_completion_chunk`